### PR TITLE
ROX-18763: PF5/FEC Upgrade

### PR DESCRIPTION
**Pull request by alwayshooin**
`rm -rf node_modules` to nuke old deps
`npm i` to run new (updated) deps
`npm run start` to run the app
click around -- everything should build and work as expected!

### CODE REVIEW

File: src/components/NotFoundMessage.tsx [hunk 1]
```diff
@@ -31,19 +32,27 @@ const NotFoundMessage: React.FunctionComponent<NotFoundMessageProps> = ({
...
 }) => {
   return (
-    <EmptyState variant={EmptyStateVariant.large} {...props}>
-      <EmptyStateIcon icon={ExclamationCircleIcon} color={dangerColor.value} />
-      <Title headingLevel="h4" size="lg">
-        {errorTitle}
-      </Title>
+    <EmptyState variant={EmptyStateVariant.lg} {...props}>
+      <EmptyStateHeader
+        titleText={<>{errorTitle}</>}
```
Comment (vjwilson) [inline src/components/NotFoundMessage.tsx:37]: Why do we need the React Fragment ( `<>...</>` ) tags here?

File: src/components/Status.js [hunk 2]
```diff
@@ -20,23 +20,23 @@ const statusMessages = {
   },
   provisioning: {
     message: statuses.provisioning,
-    component: <Spinner isSVG size="md" />,
+    component: <Spinner size="md" />,
   },
   ready: {
     message: statuses.ready,
-    component: <CheckCircleIcon className="pf-u-success-color-100" />,
+    component: <CheckCircleIcon className="pf-v5-u-success-color-100" />,
```
Comment (vjwilson) [inline src/components/Status.js:27]: Just when you thought BEM couldn't get any more awkward to read...

File: src/routes/InstancesPage/InstancesPage.js [hunk 3]
```diff
@@ -10,20 +10,21 @@ import {
...
   Spinner,
-  Title,
   Toolbar,
   ToolbarContent,
   ToolbarItem,
 } from '@patternfly/react-core';
 import {
   ActionsColumn,
-  TableComposable,
+  Table /* data-codemods */,
```
Comment (vjwilson) [inline src/routes/InstancesPage/InstancesPage.js:27]: Finally, sanity

**Review by vjwilson — APPROVED** [review]
Doesn't look too painful.

Would love to hear answers to my questions, but they don't block merge.

File: src/components/NotFoundMessage.tsx (hunk 1 above)
Comment (alwayshooin) [reply src/components/NotFoundMessage.tsx:37]: oh im not sure, i would guess the codemods threw that in there

File: package.json [hunk 4]
```diff
@@ -17,15 +17,15 @@
     "start": "HOT=true fec dev",
     "start:mock-server": "NODE_OPTIONS='--max-http-header-size=100000' node mocks/db.server.js",
     "test": "jest",
-    "postinstall": "rimraf .cache",
+    "postinstall": "ts-patch install && rimraf .cache",
```
[the remark this answers is not in the document]
Comment (alwayshooin) [reply package.json:20]: this was a requirement for the redhat insights/frontend components upgrade: 
image
https://console.stage.redhat.com/platform-docs/guides/frontend-migrations/pf-4-to-pf5

**Pull request merged**